suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -6,14 +6,14 @@ import { TokenInfo } from '@trezor/blockchain-link';
 import { EthereumTokenAccountWithBalance } from './types';
 import { isEthereumAccountSymbol } from './utils';
 
-const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
+export const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
```

Comment:
it is used in accounts import summary. We can't use selector there since account is not imported yet and we need to filter tokens there based on balance.